src/includes/classes/MenuPageOptions.php

```diff
@@ -251,9 +251,9 @@ public function __construct()
         /* --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
 
         if (IS_PRO || $this->plugin->isProPreview()) {
-            echo '<div class="plugin-menu-page-panel'.(!IS_PRO ? ' pro-preview' : '').'">'."\n";
```

Comment:
@renzms I think this line should be restored to what it was. It still applies, even though the additional class is added to the header for this panel, we still flag the entire panel as being a part of the `pro-preview`.